src/app/app.component.ts

```diff
@@ -32,13 +32,36 @@ import { VStreamState } from './shared/state/vstream/vstream.feature';
 import { VStreamActions } from './shared/state/vstream/vstream.actions';
 import { VStreamPubSubService } from './shared/services/vstream-pubsub.service';
 import { CounterCommand } from './shared/services/command.interface';
+import { AudioService } from './shared/services/audio.service';
+import { LogService } from './shared/services/logs.service';
+import { ChatService } from './shared/services/chat.service';
+import { CommandService } from './shared/services/command.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
   standalone: true,
   imports: [NavComponent, RouterOutlet],
+  providers: [
+    AzureSttService,
```

Comment:
What changed here, why the new providers here rather than in the app config?

Reply:
At I see, switched to non-root-provided.
Depending on the lifetime you want I would still rescind putting them in so config since you want to say that they're global